js/src/behaviors/snap-scroll.ts

```diff
@@ -0,0 +1,107 @@
+let snapScrollUpdating = false;
+
+/**
+ * Snap scroll behaviors to have a smooth transition from one card to another
+ * @param container the id of the snap slide container
+ * @param updateDelay default 300, the delay of button classes update when slide comes into view
```

Comment:
It doesn't seem like these two parameters are actually being used. Should `updateDelay` be tied to the setTimeout on line 93?

Reply:
good catch, we updated it but forgot to remove